ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -197,6 +197,38 @@ def test_tensorflow_Cosh(
     )
 
 
+# Cross
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(set(ivy_tf.valid_float_dtypes))
+        ),
+        num_arrays=2,
+        shared_dtype=True,
```

Comment:
so it seems like the examples being generated for Cross are not correct. It requires 2 or 3-dimensional inputs. Can you add `min_num_dims`=2 and `max_num_dims`=3?